### PR TITLE
grafana-cli: Fix installing of plugins missing directory entries in zip

### PR DESCRIPTION
**Pull request by adrianlzt**
**What this PR does / why we need it**:
If a plugin zip file does not have the base directory placed in the first position, this command will fail trying to create a file in
a directory that does not exists. Same problem if the zip file does not contain the parent directory of files/directories.

To create an unordered zip file (directory not the first one):

```
zip -r file.zip foo/bar foo/foobar foo/
```

Zip file without the base dir:

```
zip -r file.zip foo/bar foo/foobar
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/cmd/grafana-cli/commands/install_command.go [hunk 1]
```diff
@@ -233,6 +233,9 @@ func extractFiles(archiveFile string, pluginName string, filePath string, allowS
 				return fmt.Errorf(permissionsDeniedMessage, newFile)
 			}
 		} else {
+			// Create needed directories to extract file
+			os.MkdirAll(filepath.Dir(newFile), 0755)
```
Comment (aknuds1) [inline pkg/cmd/grafana-cli/commands/install_command.go:237]: The error should be checked, linting fails at the moment because it isn't.

**Review by aknuds1 — CHANGES_REQUESTED** [review]
I found a problem, but will look more closely.

File: pkg/cmd/grafana-cli/commands/install_command.go (hunk 1 above)
Comment (adrianlzt) [reply pkg/cmd/grafana-cli/commands/install_command.go]: Fixed, thanks!

**Review by aknuds1 — APPROVED** [review]
LGTM!

**Comment by aknuds1** [comment]
Thank you for contributing to Grafana!

**Pull request merged**